packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss

```diff
@@ -1,15 +1,24 @@
 @use 'sass:math';
 
 .p-chip {
-    background-color: $chipBg;
-    color: $chipTextColor;
```

Comment:
Could you please leave those SCSS variables in place, and edit their definition in [the `_variable.scss` file](https://github.com/DARPA-ASKEM/TERArium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L781-L782).